src/main/java/online/shop/db/Storage.java

```diff
@@ -2,15 +2,42 @@
 
 import java.util.ArrayList;
 import java.util.List;
+import online.shop.model.Order;
 import online.shop.model.Product;
+import online.shop.model.ShoppingCart;
+import online.shop.model.User;
 
 public class Storage {
     public static final List<Product> products = new ArrayList<>();
+    public static final List<User> users = new ArrayList<>();
+    public static final List<Order> orders = new ArrayList<>();
+    public static final List<ShoppingCart> shoppingCarts = new ArrayList<>();
     private static Long productId = 0L;
+    private static Long userId = 0L;
+    private static Long orderId = 0L;
+    private static Long shoppingCartId = 0L;
 
     public static void addProduct(Product product) {
         productId++;
         product.setId(productId);
         products.add(product);
     }
+
+    public static void addUser(User user) {
+        userId++;
+        user.setId(userId);
```

Comment:
```suggestion
        user.setId(++userId);
```
same with other methods